src/Command/FixMediaContextCommand.php

```diff
@@ -78,18 +78,7 @@ public function execute(InputInterface $input, OutputInterface $output): int
                 $this->contextManager->save($defaultContext);
             }
 
-            $defaultCategory = $this->categoryManager->getRootCategory($defaultContext);
-
-            if (!$defaultCategory) {
-                $output->writeln(sprintf(" > default category for '%s' is missing, creating one", $context));
-                $defaultCategory = $this->categoryManager->create();
-                $defaultCategory->setContext($defaultContext);
-                $defaultCategory->setName(ucfirst($context));
-                $defaultCategory->setEnabled(true);
-                $defaultCategory->setPosition(0);
-
-                $this->categoryManager->save($defaultCategory);
-            }
```

Comment:
This should be kept.
```
$categories = $this->categoryManager->getRootCategoriesForContext($defaultContext);

if ([] === $categories) {
 ...
}
```

Reply:
I don't think so: https://github.com/sonata-project/SonataClassificationBundle/blob/master/src/Entity/CategoryManager.php#L252-L295

Reply:
Whoah, it's unnatural having a method `getFoo` creating under the hood a category if there is none.
Some comment should be added in the classification, but maybe here too ?

First thing I thought here was "This call seems useless, we can remove it", but it's clearly not.

Reply:
I think the classification should clarify that it always returns the category , even if it has to create it.

Here the best IMHO would be to add integration test that ensures the category is created. But I would prefer to leave this PR without it.